dushimeemma/fs-simba-cal
# ESL-1 Setup GitHub Actions

Description

- Setup Eslint
- Setup prettier
- Configure Headwind
- Setup husky
- Setup GitHub Actions
- Setup tailwind
- Setup prisma
- Setup continuous deployment

[ ] Bug fix (Non breaking change that fixes an existing issue)

[x] New Feature (Non breaking change which adds funcitonality)

[ ] Breaking change (New feature that removes or causes an existing feature not to work as expected)

[ ] This change requires a documentation update

How Has This Been Tested?
[] Unit tests

[ ] Components Tests

[ ] Integration Tests

How Should This Be Manually tested:

- Clone this repo `git clone https://github.com/dushimeemma/fs-simba-cal/`.

- Checkout the branch `git checkout chore/ESL-1-setup-eslint`

- Run `yarn install`

- Run `yarn dev`

Checklist:
[x] My code follows all the styles and conventions outlined for this project

[x] I have linted my code before submission

[x] I have made my corresponding changes to the documentation

[x] My changes generates no new warning

[x] I have added tests that prove my feature is working or my fix is effective

[x] New and existing tests pass locally with my changes

JIRA
[ESL-1](https://www.notion.so/Setup-NextJS-project-cbf21d0431c64e9992ce10c12f2da5d9)
